### ISSUE TITLE
index.d.ts is not published to npm

### ISSUE THREAD

**Issue by jocaml** [claus/react-dat-gui#42]
I guess it's excluded so it doesn't end up in `dist/`.

**Comment by imjasonmiller** [comment]
Thanks @jocaml! As a temporary solution I added the following to my `tsconfig.json`:

```json
"compilerOptions": {
    "typeRoots": ["./src/types", "./node_modules/@types"]
}
```

I then added the code from the repo's `index.d.ts` to `.src/types/@tim-soft/react-dat-gui/index.d.ts` like so:

```ts
declare module "@tim-soft/react-dat-gui" {
   import * as React from "react"
 
   export interface DatGuiProps
   // <--snip-->
}
```
Perhaps it'll be of help to someone else.

I'm using @tim-soft's fork due to #43.

**Comment by rohan-deshpande** [comment]
Hey all, sorry for this and the delay on #43, we’d welcome PRs to get these fixed, otherwise I’ll get to them ASAP. Shouldn’t be too hard.

**Comment by anselanza** [comment]
I also need some proper TypeScript support.

**Comment by anselanza** [comment]
Implemented a simple fix here: https://github.com/claus/react-dat-gui/pull/46

**Comment by rohan-deshpande** [comment]
Resolved by #51 and #46
